src/Analysis/GWASV2/Components/DismissibleMessage/DismissibleMessage.jsx

```diff
@@ -7,8 +7,13 @@ const DismissibleMessage = ({
   title = 'Placeholder Title',
   description = 'placeholder description',
   messageType = 'success',
+  dismissMessage = null,
 }) => {
   const [open, setOpen] = useState(true);
+  const close = () => {
+    if (dismissMessage) dismissMessage();
```

Comment:
Since dismissMessage is required, do we need to check for it? I don't think you'd pass prop validation without it (line 52). Does checking for this imply there are going to be error messages that aren't added to the global state?

Reply:
It is not required. Default is `null`.